addons/account_peppol/static/src/js/portal.js

```diff
@@ -2,26 +2,25 @@
 
 import publicWidget from "@web/legacy/js/public/public_widget";
 
-publicWidget.registry.portalDetails = publicWidget.Widget.extend({
-    selector: '.o_portal_details',
-    events: {
-        'change select[name="invoice_sending_method"]': '_onSendingMethodChange',
-    },
+publicWidget.registry.portalDetails.include({
+    events: Object.assign({}, publicWidget.registry.portalDetails.prototype.events, {
+        'change select[name="invoice_sending_method"]': "_onSendingMethodChange",
```

Comment:
```suggestion
portalDetails.include({
    events: Object.assign({}, portalDetails.prototype.events || {}, {
        'change select[name="invoice_sending_method"]': "_onSendingMethodChange",
```

This would need:
`export default publicWidget.registry.portalDetails;`
and 
`import portalDetails from "@portal/js/portal";`

inspiration from payment_form.js

Reply:
@h4818 Done, Thanks.